.github/workflows/build-and-test.yml

```diff
@@ -181,8 +181,8 @@ jobs:
         # https://github.com/iotaledger/wallet.rs/pull/467/checks?check_run_id=2214108362#step:11:41
```

Comment:
Can't comment, but maybe change line 99 so it also only tests wallet?

Reply:
Also do that in https://github.com/iotaledger/wallet.rs/blob/develop/.github/workflows/private-tangle-tests.yml